VR Maze/Assets/NewBehaviourScript.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NewBehaviourScript : MonoBehaviour {
```

Comment:
This is just a default script which does nothing, can you please remove this file and its .meta file?